dmc_configs/dmc_configs/README.md

```diff
@@ -0,0 +1,176 @@
+# Experiment Configs for Key Figures
+
+Below is a summary of configs that correspond to figures in the Demonstrating Monty Capabilities paper, with descriptions motivating the choice of config parameters.
+
+**Note:** In all experiments, `use_semantic_sensor=False` should be specified. This should be the case once PR #116 is merged, updating the values for PatchViewFinderMountHabitatDatasetArgs etc to be False by default.
+
+## Figure 1 & 2: Diagramatic Figures With No Experiments
+
+## Figure 3: Robust Sensorimotor Inference
+
+Consists of 4 experiments:
+- `dist_agent_1lm` (i.e. no noise)
+- `dist_agent_1lm_noise` - Sensor noise
+- `dist_agent_1lm_randrot` - 14 random rotations  
+- `dist_agent_1lm_randrot_noise`
+
+Here we are showing the performance of the "standard" version of Monty, using:
+- 77 objects
+- Goal-state-driven/hypothesis-testing policy active
+- A single LM (no voting)
+
+The main output measure is accuracy and rotation error as a function of noise conditions.
+
+## Default Parameters for Figures 4+
+Unless specified otherwise, the following figures/experiments use:
+- 77 objects
+- 14 random rotations
```

Comment:
Didn't we say we will test on 5 random rotations last time?

Reply:
Good catch